src/schema/v2/conversation/index.ts

```diff
@@ -463,7 +469,21 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
     messagesConnection,
     unread: {
       type: GraphQLBoolean,
-      description: "True if there is an unread message by the user.",
+      description: "True if there is an unread message by the Collector(from).",
+      deprecationReason: "Use `unreadByCollector` instead",
+      resolve: (conversation) => {
+        const memoizedLastMessageId = lastMessageId(conversation)
+        const { from_last_viewed_message_id } = conversation
+        return (
+          !!from_last_viewed_message_id &&
+          !!memoizedLastMessageId &&
+          from_last_viewed_message_id < memoizedLastMessageId
+        )
+      },
+    },
```

Comment:
#non-blocking suggestion: maybe move this resolver into a shared function and then add a test for that function? Not sure if it makes sense or will ergonomically feel right, just thinking that there's enough shared data between the two that we could unify / test.

Reply:
totally, check it out please https://github.com/artsy/metaphysics/pull/4952/commits/ecaa29a40857c400a175443d0dc8a2b198bea7dd 
any further requests I can do in follow-up PRs